frontend/packages/ceph-storage-plugin/src/components/kms-config/vault-config.tsx

```diff
@@ -0,0 +1,239 @@
+import * as React from 'react';
```

Comment:
If required, we can skip review of this file `frontend/packages/ceph-storage-plugin/src/components/kms-config/vault-config.tsx`.
This is old code, just changed the file location and file name, hence git is showing it as a new file.